diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter4/NullChecksRule.kt

```diff
@@ -185,7 +184,6 @@ class NullChecksRule(configRules: List<RulesConfig>) : DiktatRule(
 
     private fun warnAndFixOnNullCheck(
         condition: KtBinaryExpression,
-        canBeAutoFixed: Boolean,
```

Comment:
IF this parameter was intended as a part of this rule's API for future usage, we shouldn't remove it just because of compiler warning. It can be suppressed. @akuleshov7 do you remember why was this parameter needed?

Reply:
I suppose, when we need this parameter, we'll add it